src/components/RelicModal.jsx

```diff
@@ -331,174 +843,481 @@ export default function RelicModal(props) {
+                      <RadioIcon value={Constants.Parts.Head} src={Assets.getPart(Constants.Parts.Head)} />
+                      <RadioIcon value={Constants.Parts.Hands} src={Assets.getPart(Constants.Parts.Hands)} />
+                      <RadioIcon value={Constants.Parts.Body} src={Assets.getPart(Constants.Parts.Body)} />
+                      <RadioIcon value={Constants.Parts.Feet} src={Assets.getPart(Constants.Parts.Feet)} />
+                      <RadioIcon value={Constants.Parts.PlanarSphere} src={Assets.getPart(Constants.Parts.PlanarSphere)} />
+                      <RadioIcon value={Constants.Parts.LinkRope} src={Assets.getPart(Constants.Parts.LinkRope)} />
+                    </Radio.Group>
+                  </Form.Item>
+
+                  <HeaderText>Set</HeaderText>
+                  <Form.Item size="default" name="set">
+                    <Select
+                      showSearch
+                      allowClear
+                      style={{
+                        width: 300,
+                      }}
+                      placeholder="Sets"
+                      options={setOptions}
+                      maxTagCount="responsive"
+                    >
+                    </Select>
+                  </Form.Item>
+                  <HeaderText>Enhance / Grade</HeaderText>
+
+                  <Flex gap={10}>
+                    <Form.Item size="default" name="enhance">
+                      <Select
+                        showSearch
+                        style={{ width: 115 }}
+                        options={enhanceOptions}
+                      />
+                    </Form.Item>
+
+                    <Form.Item size="default">
+                      <Button style={{ width: 50 }} onClick={plusThree}>
+                        +3
+                      </Button>
+                    </Form.Item>
+
+                    <Form.Item size="default" name="grade">
+                      <Select
+                        showSearch
+                        style={{ width: 115 }}
+                        options={[
+                          { value: 2, label: '2 ★' },
+                          { value: 3, label: '3 ★' },
+                          { value: 4, label: '4 ★' },
+                          { value: 5, label: '5 ★' },
+                        ]}
+                      />
+                    </Form.Item>
+                  </Flex>
+                </Flex>
+              </Col>
+              <Col span={8}>
+                <Flex vertical gap={5}>
+                  <HeaderText>Equipped by</HeaderText>
+                  <Form.Item size="default" name="equippedBy">
+                    <Select
+                      showSearch
+                      filterOption={filterOption}
+                      style={{ width: 150, height: 35 }}
+                      options={characterOptions}
+                      onChange={(e) => {
+                        setWearerID(relicForm.getFieldValue('equippedBy'))
```

Comment:
Form.useWatch kinda does what you're trying to implement here https://ant.design/components/form#formusewatch so we can hook into the form events and extract a value to display

Reply:
useWatch wouldn't really work since I needed to do more than just match the text